tools/doc/node_modules/js-yaml/index.js

```diff
@@ -1,15 +1,7 @@
 'use strict';
 
-// Hack to load the js-yaml module from eslint.
```

Comment:
unrelated change?

Reply:
grr... this one is becoming quite annoying.